samples/highcharts/demo/stellar-chart/demo.js

```diff
@@ -0,0 +1,232 @@
+        }
+    }, {
+        type: 'pie',
+        dataLabels: {
+            enabled: false
+        },
+        size: '40%',
+        innerSize: '85%',
+        zIndex: -1,
+        point: {
+            events: {
+                mouseOver() {
+                    const {
+                            minDate,
+                            maxDate
+                        } = this.options,
+                        point = this,
+                        series = this.series,
+                        chart = series.chart,
+                        bubbleSeries = chart.series[0];
+
+                    bubbleSeries.points.forEach(point => {
+                        if (
+                            point.options.custom.discoveryDate < minDate ||
+                            point.options.custom.discoveryDate >= maxDate
+                        ) {
+                            point.graphic.attr({
+                                opacity: 0.2
+                            });
+                        }
+                    });
+
+                    series.customLabel = fillCenter(point.percentage,
+                        point.options.custom.info, chart, series.customLabel);
+                },
+                mouseOut() {
+                    const chart = this.series.chart,
+                        series = this.series,
+                        bubbleSeries = chart.series[0];
+
+                    bubbleSeries.points.forEach(point => {
+                        point.graphic.attr({
+                            opacity: 1
+                        });
+                    });
+                    series.customLabel = fillCenter(100, '1990s-2020s', chart,
+                        series.customLabel);
+                }
+            }
+        },
+
+        data: [{
+            y: 12,
+            color: '#6CDDCA',
+            custom: {
+                info: '1990s',
+                minDate: 1990,
+                maxDate: 2000
+            }
+        },
+        {
+            y: 47,
+            color: '#C771F3',
+            custom: {
+                info: '2000s',
+                minDate: 2000,
+                maxDate: 2010
+            }
+        },
+        {
+            y: 117,
+            color: '#4D90DB',
+            custom: {
+                info: '2010s',
+                minDate: 2010,
+                maxDate: 2020
+            }
+        },
+        {
+            y: 64,
+            color: '#FAB776',
+            custom: {
+                info: '2020s',
+                minDate: 2020,
+                maxDate: 2030
+            }
+        }
+        ]
+    }]
+});
+
+function fillCenter(percentage, info, chart, customLabel) {
+    const labelText = `<span style='font-size: 2em'>☉</span><span
+    class='uppercase'><br><span class='decade'>${info}</span><br>
```

Comment:
Can we leave the decade out of the uppercase class? Decades are commonly written 1900s, 2000s, so the upper case S makes it kind of unfamiliar.